webpack/assets/javascripts/react_app/components/DiffView/DiffView.stories.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```

Comment:
did you publish the storybook somewhere?

Reply:
[Storybook](https://glekner.github.io/foreman/?knob-Old%20Text=Old%20Text&knob-New%20Text=New%20Text&selectedKind=Components%2FDiffView&selectedStory=DiffView&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs) @terezanovotna